package.json

```diff
@@ -73,54 +67,35 @@
     "@types/offline-js": "^0.7.28",
     "@types/rangy": "0.0.30",
     "@types/webpack-env": "^1.16.0",
-    "ajv": "^6.10.0",
-    "angular-mocks": "^1.8.2",
-    "babel-core": "^6.26.3",
-    "browserstack-local": "^1.4.0",
     "copy-webpack-plugin": "^8.1.0",
     "core-js": "^3.6.5",
```

Comment:
note to self: core-js appears twice in the package.json file!  Oops.  I'm confused now and I'm guessing that webpack still works because core-js is included in dev dependencies.